src/libraries/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs

```diff
@@ -23,7 +23,7 @@ public static string ToServerString(this Cookie cookie)
 
         public static Cookie Clone(this Cookie cookie)
         {
-            s_cloneFunc ??= (Func<Cookie, Cookie>)typeof(Cookie).GetMethod("Clone", BindingFlags.Instance | BindingFlags.NonPublic)!.CreateDelegate(typeof(Func<Cookie, Cookie>));
+            s_cloneFunc ??= (Func<Cookie, Cookie>)typeof(Cookie).GetMethod("Clone", BindingFlags.Instance | BindingFlags.NonPublic)?.CreateDelegate(typeof(Func<Cookie, Cookie>))!;
```

Comment:
why would these methods not exist?

Reply:
If there were any expected circumstances for that, there should be proper error handling, not just asserts. But asserts are here so they deserve a chance to hit.